src/pages/Todopage/Todopage.element.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from "styled-components";
+
+interface TPProps {
+  hoverRender?: boolean;
+  render: boolean;
+}
+
+export const TodopageContainer = styled.div<TPProps>`
+  width: ${(props) => (props.render ? "max(300px, 30vw)" : "0")};
+  height: ${(props) => (props.render ? "max(500px, 50vw)" : "0")};
+  position: ${(props) => (props.render ? "initial" : "fixed")};
+  margin: 0 auto;
+  background-color: #1e1e1e;
+  border-radius: 12px;
+  z-index: 2;
+  visibility: ${(props) => (props.render ? "visible" : "hidden")};
+  box-shadow: 0 0 10px 0px #000000;
+  transition: width 0.2s, height 0.2s, visibility 0.05s linear 0.15s;
+`;
+
+export const CardToolBar = styled.div<TPProps>`
+  display: flex;
+  align-items: center;
+  height: 40px;
+  padding: 12px;
+  background-color: #312f34;
+  border-radius: 12px 12px 0 0;
+  box-sizing: border-box;
+  position: relative;
+  visibility: ${(props) => (props.render ? "visible" : "hidden")};
+`;
+
+export const CardPlusBtn = styled.div`
+  width: 20px;
+  height: 20px;
+  color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 24px;
+  border-radius: 4px;
+  position: absolute;
+  right: 12px;
+  padding: 2px;
+  &:hover {
+    background-color: #656565;
+    cursor: url(Link.cur) 0 0, pointer;
+  }
+  &::before {
+    content: "+";
+  }
+`;
+
+export const NotHoverDiv = styled.div<TPProps>`
+  opacity: ${(props) => (props.hoverRender ? 0 : 1)};
+  display: ${(props) => (props.render ? "flex" : "none")};
+  flex-direction: column;
+  height: 100%;
+  transition: opacity 0.1s;
+`;
+
+export const CardMainDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: calc(100% - 40px);
+  border-radius: 0 0 12px 12px;
```

Comment:
```suggestion
  border-radius: 0 12px;
```
요건 이렇게 줄여쓸 수 있을 것 같아요!